templates/schematron.xspec

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+    * Update /x:description/@schematron which should point to the Schematron file being tested.
+    * Write your test scenario.
+      See https://github.com/xspec/xspec/wiki/Writing-Scenarios-for-Schematron#writing-tests for details.
+-->
+<x:description xmlns:x="http://www.jenitennison.com/xslt/xspec"
+    schematron="path/to/my-schematron.sch">
+
+    <x:scenario label="Scenario for testing a rule">
+        <x:context>
+            <example>XML fragment to be validated by the Schematron being tested</example>
```

Comment:
Does it make sense to do something like `<example><section>XML fragment to be validated by the Schematron being tested</section></example>` so that the reference to `/section[1]` on line 15 fits in? I know the template isn't expected to run out of the box, but it could be a little confusing that the location doesn't map to anything. If you want to keep this as simple as possible, maybe go the other way and remove `/section[1]` from line 15. Or do nothing and keep what you have. :)

Reply:
I was thinking about that too and you hit the nail on it. Added `<section>` in 2b58324678b53083e20ce75586feb1a49135a759.